### DISCUSSION TITLE
@quasar/app-vite: Missing! → 1.1.3

### DISCUSSION THREAD

**Discussion by paulsingh** [quasarframework/quasar#15200] [General - Components / Directives / etc] [answered]
When running `quasar upgrade`, I get the following output:

> $ quasar upgrade
> 
>   Quasar CLI · Gathering information with npm...
> 
>  @quasar/app-vite: Missing! → 1.1.3
> 
>  See https://quasar.dev/start/release-notes for release notes.
>  Run "quasar upgrade -i" to do the actual upgrade.

So then I run `quasar upgrade -i` and get the following output:

> $ quasar upgrade -i
> 
>   Quasar CLI · Gathering information with npm...
> 
>  @quasar/app-vite: Missing! → 1.1.3
> 
>   Quasar CLI · Running "npm install --save-dev @quasar/app-vite@^1.1.3"
> 
> 
> up to date, audited 39 packages in 5s
> 
> 7 packages are looking for funding
>   run `npm fund` for details
> 
> found 0 vulnerabilities
> 
>   Quasar CLI · Successfully upgraded Quasar packages.
> 
> $ 

I've tried removing `node_modules` and `package-lock.json` but keep ending up at the same spot. 

Has anyone else seen this lately? 

In case it's helpful, here's the output of `quasar info`:

> $ quasar info
> 
> Operating System   	Darwin(22.2.0) - darwin/x64
> NodeJs             	16.14.0
> 
> Global packages    	
>   NPM              	9.2.0
>   yarn             	1.22.19
>   @quasar/cli      	1.3.2
>   @quasar/icongenie	2.5.3
>   cordova          	Not installed
>

**Comment by rstoenescu** [comment, marked as answer]
I think this is related to the PNPM support in the global installation of Quasar CLI (q/cli) which deals with the "upgrade" command. Released today a fix for it (q/cli v1.4.0).

**Comment by paulsingh** [comment]
Unfortunately, doesn't seem to resolve the problem. First, upgraded q/cli via `npm i -g @quasar/cli`:

> $ npm i -g @quasar/cli
> 
> changed 358 packages in 4s
> 
> 45 packages are looking for funding
>   run `npm fund` for details

Confirmed that I'm running the latest version:

> $ quasar -v
> 1.4.0

Then ran `quasar upgrade -i`

> $ quasar upgrade -i
> 
>   Quasar CLI · Gathering information with npm...
> 
>  @quasar/app-vite: Missing! → 1.1.3
> 
>   Quasar CLI · Running "npm install --save-dev @quasar/app-vite@^1.1.3"
> 
> up to date, audited 39 packages in 889ms
> 
> 7 packages are looking for funding
>   run `npm fund` for details
> 
> found 0 vulnerabilities
> 
>   Quasar CLI · Successfully upgraded Quasar packages.

And then, to double check, ran `quasar upgrade` once more:

> $ quasar upgrade
> 
>   Quasar CLI · Gathering information with npm...
> 
>  @quasar/app-vite: Missing! → 1.1.3
> 
>  See https://quasar.dev/start/release-notes for release notes.
>  Run "quasar upgrade -i" to do the actual upgrade.

**Reply by rstoenescu** [reply]
Check your project folder. Something is really off with it since packages get installed but they are still being reported as missing. The upgrade algorithm is bulletproof and battle tested by now. A reproduction repo can help us help you.
